ios/CodePush/CodePush.m

```diff
@@ -61,7 +62,11 @@ @implementation CodePush {
 
 + (NSURL *)binaryBundleURL
 {
-    return [[NSBundle mainBundle] URLForResource:bundleResourceName withExtension:bundleResourceExtension];
+    NSBundle *bundle = [NSBundle mainBundle];
+    if (isAppExtension && [[bundle.bundleURL pathExtension] isEqualToString:@"appex"]) {
+      bundle = [NSBundle bundleWithURL:[[bundle.bundleURL URLByDeletingLastPathComponent] URLByDeletingLastPathComponent]];
```

Comment:
Is this line trying to reference the containing app's bundle from within the running extension? If so, could you add a comment explaining that? Presumably this logic could break in future iOS versions, so I'd love to be explicit about what we're trying to achieve if there isn't a better solution.